src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/models/rgw-storage-class.model.ts

```diff
@@ -95,6 +97,13 @@ export interface PlacementTarget {
   };
 }
 
+export interface PlacementTarget {
```

Comment:
Whats the difference from above ? `PlacementTargets`
Ideally types should not be defined like this.